app/scripts/models/account.js

```diff
@@ -96,7 +96,8 @@ define([
             self.set('verified', verified);
           }, function () {
             // Ignore errors; we'll just fetch again when needed
-          });
+            // TODO Remove once https://github.com/eslint/eslint/issues/3279 gets fixed
```

Comment:
But the issue was updated 13 days ago, and eslint is a very happening community, so I think it should be fixed in the next release. But yeah, other than these indentation stuff we  dont really have any other issues updating, and bithound will look better :P :D